sunflower-land/sunflower-land
# [FIX] check for undefined values in seed `stock`

# Description

Fixes `undefined in stock` value in seeds whenever they get sold out and the user reloads the game.

![image](https://user-images.githubusercontent.com/50954098/153746839-13cfcc77-59d5-4b7d-aec9-cb01a7d79c68.png)

## Type of change

Please delete options that are not relevant.


# How Has This Been Tested?

steps to reproduce:
1. Buy up all the stocks of a seed. 
2. Save the game _(but do not sync on chain yet)_
2. Reload the game, you should see the value in stock is `undefined`

![sfl-undefined-stock](https://user-images.githubusercontent.com/50954098/153747021-545835ee-c634-4e92-87fe-4e4039622fed.gif)



**after the fix**:
![image](https://user-images.githubusercontent.com/50954098/153747098-e4d2258a-15ff-4c95-8629-e79d37193980.png)


# Checklist:
